src/Utils/__test__/logger.test.ts

```diff
@@ -0,0 +1,65 @@
+import { sendErrorToService } from "Utils/errors"
+import createLogger from "Utils/logger"
+
+jest.mock("Utils/errors")
+
+describe("logger", () => {
+  describe("#createLogger", () => {
+    let logger
+    beforeEach(() => {
+      console.log = jest.fn()
+      console.warn = jest.fn()
+      console.error = jest.fn()
+      logger = createLogger("testing")
+    })
+
+    describe("#log", () => {
+      it("logs given statements with console.log", () => {
+        logger.log("msg")
+        expect(console.log).toBeCalledWith("testing |", "msg", "\n")
+      })
+    })
+
+    describe("#warn", () => {
+      it("logs given warnings with console.warn", () => {
+        logger.warn("msg")
+        expect(console.warn).toBeCalledWith("testing |", "msg", "\n")
+      })
+    })
+
+    describe("#error", () => {
+      describe("when it should not capture errors", () => {
+        it("does not send errors to service", () => {
+          logger.error(new Error("msg"))
+          expect(sendErrorToService).not.toHaveBeenCalled()
+        })
+      })
+
+      describe("when it should capture errors", () => {
+        let originalEnv
+        beforeAll(() => {
+          originalEnv = process.env
+          process.env = Object.assign({}, originalEnv, {
+            NODE_ENV: "staging",
+          })
+        })
+
+        afterAll(() => {
+          process.env = originalEnv(sendErrorToService as any).mockReset()
```

Comment:
Isn't the original environment just a hash? What is `.mockReset()` for?

Reply:
oh! That's definitely a typo, I think from prettier. I'll fix that.